chatkit-core/src/main/kotlin/com/pusher/chatkit/SynchronousChatManager.kt

```diff
@@ -23,10 +24,11 @@ import com.pusher.util.asSuccess
 import com.pusher.util.orElse
 import elements.Error
 import elements.Errors
+import java.lang.IllegalStateException
 import java.util.concurrent.CountDownLatch
 
 class SynchronousChatManager constructor(
-    private val instanceLocator: String,
+    internal val instanceLocator: String,
```

Comment:
Why the increased access?

Reply:
We use it in: https://github.com/pusher/chatkit-android/pull/85/files#diff-8a7a9ee39e6c725e06a1363e991de1d7R201:
```
pn.start(client.platformInstance.id, chatManager.beamsTokenProviderService)
```

Internal will make it visible to the same module (internal for the SDK), so end-users will not have access to it.